### PR TITLE
Balance changes and bug fixes

### PR DESCRIPTION
**Pull request by modargo**
Leaving this for you to take a quick look over and merge -- we discussed all the balance changes, but there are enough changes that it makes sense.

### Balance changes
Gems pack: change Citrine to rare
Rip pack: reduce Fragile Shrug block by 1
Rip pack: increase Inspiration cost by 1
Utility pack: increase Greater Hex cost by 1
Go With The Flow pack: change Flow to draw next turn equal to half the number of discarded cards (instead of equal to the number of discarded cards)
Go With The Flow pack: increase Hydraulics damage by 2
Go With The Flow pack: increase Aqueducts damage by 1
Creativity pack: reduce Makeshift Shield block by 1
Creativity pack: change Souvenirs upgrade to upgrade the created card (instead of creating an additional card)
Creativity pack: change Striking Strike to cost 2 and to upgrade to cost 1

### Fixes
Creativity pack: make the animation for Mimicry more consistent with other cards that discover a new card
Monster Hunter pack: fix Hunting Instincts showing the wrong block amount when you hold the card and hover it over the playing area
Replicators pack: fix Iterative Design being incorrectly marked as turn-based
Utility pack: tweak text formatting for Lesser Hex and Greater Hex to have each effect on its own line
Legacy pack: add keyword highlighting for Slow

### CODE REVIEW

File: src/main/java/thePackmaster/cards/creativitypack/Mimicry.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public void update() {
                     }
                     return o;
                 }).collect(Collectors.toCollection(ArrayList::new));
-                addToBot(new FlexibleDiscoveryAction(JediUtil.createCardsForDiscovery(tmpGrp), false));
+                addToTop(new FlexibleDiscoveryAction(JediUtil.createCardsForDiscovery(tmpGrp), false));
```
Comment (modargo) [inline src/main/java/thePackmaster/cards/creativitypack/Mimicry.java:50]: With this change, the card stays up in the background while you're doing the discovery action, which is the same way it works for Wish/Discovery/etc. Need to use addToTop to achieve that due to nested actions.

I'm so used to some aspects of how the base game does things that stuff like this is disconcerting to me if I see it a lot. That's part of how I notice small consistency issues like this.

File: src/main/java/thePackmaster/cards/monsterhunterpack/HuntingInstincts.java [hunk 2]
```diff
@@ -27,8 +27,8 @@ public void triggerOnGlowCheck() {
     }
 
     @Override
-    public void applyPowers() {
-        super.applyPowers();
+    public void applyPowersToBlock() {
```
Comment (modargo) [inline src/main/java/thePackmaster/cards/monsterhunterpack/HuntingInstincts.java:30]: The issue with the previous code was that `applyPowers` was changing the block, but `calculateCardDamage` wasn't. Since `calculateCardDamage` overwrites damage/block values when a card either acquires a target or is played, this meant that the numbers showed up wrong when the card acquired a target (and it's why the use action needed a `hasHuntTarget` check).

Overriding both `applyPowers` and `calculateCardDamage` would also fix this, but the nice thing about `applyPowersToBlock` is that it gets called by both those methods. Convenient if you're doing something that only affects block.

File: src/main/java/thePackmaster/cards/rippack/Inspiration.java [hunk 3]
```diff
@@ -22,17 +22,15 @@ public class Inspiration extends AbstractRipCard {
...
+        super(ID, 1, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.NONE);
         exhaust = true;
         CardModifierManager.addModifier(this, new RippableModifier());
     }
 
     @Override
     public void applyPowers() {
         super.applyPowers();
-        int statusAndArtCardsInExhaust = AbstractDungeon.player.exhaustPile.group.stream().filter(card -> isArtCard(card) || card.type == CardType.STATUS).collect(Collectors.toList()).size();
+        int statusAndArtCardsInExhaust = (int) AbstractDungeon.player.exhaustPile.group.stream().filter(card -> isArtCard(card) || card.type == CardType.STATUS).count();
```
Comment (modargo) [inline src/main/java/thePackmaster/cards/rippack/Inspiration.java:33]: Couldn't resist some drive-by simplifcation.

File: src/main/java/thePackmaster/powers/replicatorspack/IterativeDesignPower.java [hunk 4]
```diff
@@ -26,7 +26,7 @@ public class IterativeDesignPower extends AbstractPackmasterPower implements Clo
 
 
     public IterativeDesignPower(AbstractCreature owner, int amount) {
-        super(POWER_ID, NAME, AbstractPower.PowerType.BUFF, true, owner, amount);
+        super(POWER_ID, NAME, AbstractPower.PowerType.BUFF, false, owner, amount);
```
Comment (modargo) [inline src/main/java/thePackmaster/powers/replicatorspack/IterativeDesignPower.java:29]: My crusade continues.

File: src/main/resources/anniv5Resources/localization/eng/utilitypack/Cardstrings.json [hunk 5]
```diff
@@ -32,12 +32,12 @@
   },
   "${ModID}:LesserHex": {
     "NAME": "Lesser Hex",
-    "DESCRIPTION": "Apply !M! Weak and !M! Vulnerable. NL Enemy loses !M! Strength. NL Add a *Greater\u00a0Hex to your discard pile. NL Exhaust.",
+    "DESCRIPTION": "Apply !M! Weak. NL Apply !M! Vulnerable. NL Enemy loses !M! Strength. NL Add a *Greater\u00a0Hex to your discard pile. NL Exhaust.",
```
Comment (modargo) [inline src/main/resources/anniv5Resources/localization/eng/utilitypack/Cardstrings.json:35]: This change makes the card text look better and matches how Uppercut is written. AlexMdle pointed it out to me.

**Comment by erasels** [comment]
Looks great
